src/DETHRACE/pc-dos/dossys.c

```diff
@@ -308,7 +308,8 @@ void PDInitialiseSystem() {
 // IDA: void __cdecl PDShutdownSystem()
 void PDShutdownSystem() {
     LOG_TRACE("()");
-    NOT_IMPLEMENTED();
+
+    exit(0);
```

Comment:
Not needed now, but perhaps we need to add a harness hook here?
We can then destroy the OpenGL and SDL contexts.

Reply:
good point, yes, added it 👍